src/labels/CustomLabels.labels

```diff
@@ -6003,6 +6557,30 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>stgDocumentation</shortDescription>
         <value>Documentation</value>
     </labels>
+    <labels>
+        <fullName>stgDeploymentCustomMetadataMappingEnabled</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgDeploymentCustomMetadataMappingEnabled</shortDescription>
+        <value>Custom Metadata mapping is enabled.</value>
+    </labels>
+    <labels>
+        <fullName>stgDeploymentHasInvalidHelpTextMappings</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>stgDeploymentHasInvalidHelpTextMappings</shortDescription>
+        <value>Advanced Mapping is enabled, but we found some invalid Help Text mappings that couldn't migrate. &lt;b&gt;Take a screenshot of the table below, it will not be displayed again after you leave this page.&lt;/b&gt;</value>
```

Comment:
"couldn't migrate" -> "couldn't be migrated"? Comma in the second sentence should be a semicolon, I believe.

Reply:
```suggestion
        <value>Advanced Mapping is enabled, but we found some invalid Help Text mappings that couldn't be migrated. &lt;b&gt;Take a screenshot of the table below; it will not be displayed again after you leave this page.&lt;/b&gt;</value>
```